### ISSUE TITLE
Logging of failedInput in exceptions

### ISSUE THREAD

**Issue by mkonietzka** [barend/java-iban#14]
Some of the exception classes like eg
```
public class UnknownCountryCodeException extends IllegalArgumentException {
    private final String failedInput;

    UnknownCountryCodeException(String failedInput) {
        super("Unknown country code in " + failedInput);
        this.failedInput = failedInput;
    }

    public String getFailedInput() {
        return failedInput;
    }
}
```
uses the `failedInput` in the exception message. In environments with an automated logging of exception messages this will lead to print out IBANs in log files. IBANs are in most cases sensitive data, you will not print in log files. 

Is there a possibility that you are no longer using `failedInput' in the error message so that the IBAN no longer appears in logfiles?

**Comment by barend** [comment]
If any of these exceptions fire, then it's not a valid IBAN, and thus it cannot be sensitive data because it's simply alphabet soup 😉. Of course, in practice, this will often be something that's _really close_ to valid. I'll scrub the bad input from the exception messages and make a point release soon. Thank you for opening an issue.

**Comment by barend** [comment]
@mkonietzka I have published release 1.6.1 to Maven Central.
